### ISSUE TITLE
Allow `opa fmt` action on a given file

### ISSUE THREAD

**Issue by anderseknert** [open-policy-agent/opa-idea-plugin#147]
# What would you like to be added

Similar to how the Go plugin adds an action to run the formatter on a file, it would be great if we provided one for `opa fmt`.

**Comment by vgramer** [comment]
agreed it may a bit tedious to have to file open and have the focus to reformat it...
